app/retail/templates/shared/head.html

```diff
@@ -39,13 +39,16 @@
     <!-- Forms -->
     <link rel="stylesheet" href="{% static "v2/css/forms/button.css" %}" />
     <link rel="stylesheet" href="{% static "v2/css/forms/checkbox.css" %}" />
+    <link rel="stylesheet" href="{% static "v2/css/forms/drop.css" %}" />
     <link rel="stylesheet" href="{% static "v2/css/forms/form.css" %}" />
     <link rel="stylesheet" href="{% static "v2/css/forms/input.css" %}" />
     <link rel="stylesheet" href="{% static "v2/css/forms/label.css" %}" />
     <link rel="stylesheet" href="{% static "v2/css/forms/radio.css" %}" />
     <link rel="stylesheet" href="{% static "v2/css/forms/select.css" %}" />
     <link rel="stylesheet" href="{% static "v2/css/external_bounties/checkboxes.css" %}" />
 
+    <link rel="stylesheet" href="{% static "v2/css/grants/grant.css" %}" />
```

Comment:
@KennethAshley wanna move this into grant pages, rather than keeping in in the header?